### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by ThamaraHessel**
Allows  to standardize the environment to run the
project without installing local dependencies,
such as Jekyll.

Use the README to follow the instructions.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+## [PHPWomen Br](http://phpwomen.org.br) site
 
-O tema escolhido pelo grupo Arcana by HTML5 UP
+#### Getting Started
+  - Cloning the repository
+    ```$ git clone git@github.com:phpwomenbr/site.git ```
+  - Enter into folder
+    ```$ cd site/ ```
+  - Setup the environment
+    ```$ docker-compose up ```
```
Comment (ThamaraHessel) [inline README.txt]: Wowwww the docker-compose turns the things more easy =)

**Comment by alganet** [comment]
I could not follow the instructions :( My jekyll server returns a directory index:

screen shot 2017-08-23 at 14 24 49

**Comment by ThamaraHessel** [comment]
@alganet, rsrsrsrs, my last change broke this, I will fix it

**Comment by ThamaraHessel** [comment]
@alganet "agora vai" please, try again guy ;)

**Pull request merged**